src/PowerSystems.jl

```diff
@@ -387,6 +387,20 @@ export get_renewable_unit
 export get_interconnection_rating
 export get_interconnection_impedance
 
+# Subsystems
+export add_subsystem!
+export get_subsystems
+export get_num_subsystems
```

Comment:
lets use a full word  here.

Reply:
`get_number_of_subsystems` is quite verbose. We have several get_num_* functions already.
```
get_num_components
get_num_members
get_num_steps
get_num_supplemental_attributes
get_num_time_series
get_num_time_series_by_type
```
We do have this one also: `get_number_of_dimensions`.

Reply:
ok fine